drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -354,6 +375,10 @@ static int sdw_compute_bus_params(struct sdw_bus *bus)
 		clk_buf = NULL;
 	}
 
+	/* If dynamic scaling is supported, don't try higher freq */
```

Comment:
logical inversion in comment, should be "NOT supported"?

Reply:
Fixed.